packages/core/src/types/configs.ts

```diff
@@ -22,8 +22,8 @@ export interface ConfigRuleDefinitionObject {
 }
 
 export interface ConfigUseDefinition {
-	files?: AnyLevelDeep<FilesValue>;
-	rules?: AnyLevelDeep<ConfigRuleDefinition>;
+	files: AnyLevelDeep<FilesValue>;
+	rules: AnyLevelDeep<ConfigRuleDefinition>;
```

Comment:
I don't know why these were optional before. They really should be required; there's no reason why a user would want to omit either.